docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,154 @@
+# How to integrate Autoware with your vehicle
+
+This page demonstrates how to integrate Autoware with a real vehicle.
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- Onboard computer that satisfies the prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
+- The following devices attached
+  - Actuator
+  - LiDAR
+  - Optional: Inertial measurement unit
+  - Optional: Camera
+  - Optional: GNSS
+
+## 2. Create maps
+
+You need both a pointcloud and a vector map to take full advantage of Autoware.
+Since mapping algorithm such as SLAM (simultaneous localization and mapping) is not implemented in the current Autoware, you may need to use 3rd party tools for this step.
+
+### Create a pointcloud map
+
+Use 3rd party tools such as the LiDAR-based SLAM package to create a pointcloud map. Autoware supports the `.pcd` format for this map.
+
+### Create vector map
+
+Autoware supports lanelet2 format for a vector map. Use 3rd party tools or [Vector Map Builder](https://tools.tier4.jp/) to get the `.osm` file.
+
+## 3. Create your meta-repository
+
+A recommended way to integrate Autoware with your real vehicle is to create a meta-repository for the vehicle.
+Create a forked repository of [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) (we refer to this as meta-repository) and clone the repository.
+
+```bash
+git clone git@github.com:YOUR_NAME/autoware.YOURS.git
+```
+
+## 4. Create the description packages of your vehicle
+
+Next, you need to create description packages that define the vehicle and sensor configuration of your vehicle.
+Once it is done, you can launch your vehicle model by specifying `vehicle_model:=YOUR_VEHICLE` `sensor_model:=SAMPLE_SENSOR_KIT` in the autoware launchers.
+
+Create the following two packages:
+
+- YOUR_VEHICLE_launch (see [here](https://github.com/autowarefoundation/sample_vehicle_launch) for example)
+- YOUR_SENSOR_KIT_launch (see [here](https://github.com/autowarefoundation/sample_sensor_kit_launch) for example)
+
+It is recommended you write the above two packages in `autoware.repos` file of your meta-repository.
+
+### Adapt YOUR_VEHICLE_launch for autoware launching system
+
+#### At YOUR_VEHICLE_description
+
+Define URDF and parameters in the package (see [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) for example).
```

Comment:
[The Google Developer Documentation Style Guide recommends using meaningful link text](https://developers.google.com/style/link-text#write-link-text) (I have been guilty of doing this too 😅)

```suggestion
Define URDF and parameters in the vehicle description package (refer to the [sample vehicle description package](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) for an example).
```